staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -231,16 +232,9 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 	for e := range errCh {
 		errs = append(errs, e)
 	}
-	if len(errs) == 0 {
-		return nil
-	}
-	if len(errs) > 1 {
-		for i := 1; i < len(errs); i++ {
-			// TODO: merge status errors; until then, just return the first one.
-			utilruntime.HandleError(errs[i])
-		}
-	}
-	return errs[0]
+	err = utilerrors.NewAggregate(errs)
```

Comment:
I don't think this works. The errs are a bunch of [status error](https://github.com/hzxuzhonghu/kubernetes/blob/cc89b775ac02cdaaa68f4db6b6ab522a98c78cfb/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go#L272), we want to keep it a status error after aggregated, so it needs more careful aggregation.

Reply:
Ok, I will check. Thanks

Reply:
@caesarxuchao I checked, and find it not valid using `utilerrors.NewAggregate(errs)`.